ai/integrations/anthropic/anthropic_oss_example.ipynb

```diff
@@ -0,0 +1,302 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "3d9f39e4-2a86-45bd-abac-98117aefa7ac",
+   "metadata": {},
+   "source": [
+    "# Anthropic Tool calling capabilities with OSS Unity Catalog\n",
+    "\n",
+    "## Prerequisites\n",
+    "\n",
+    "**API Key**\n",
+    "To run this tutorial, you will need an Anthropic API key. For testing purposes, you can generate a new account, and use your evaluation test key (no credit card required!).\n",
+    "\n",
+    "Once you have acquired your key, set it to the environment variable `ANTHROPIC_API_KEY`.\n",
+    "\n",
+    "Below, we validate that this key is set properly in your environment.\n",
+    "\n",
+    "**Packages**\n",
+    "\n",
+    "To interface with both UnityCatalog and Anthropic, you will need to install the following packages:\n",
+    "\n",
+    "```shell\n",
+    "pip install anthropic unitycatalog-ai unitycatalog-anthropic unitycatalog-client\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "580c0b0f-3ab8-4d5a-9dea-0ee8c516c566",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "assert (\n",
+    "    \"ANTHROPIC_API_KEY\" in os.environ\n",
+    "), \"Please set the ANTHROPIC_API_KEY environment variable to your Anthropic API key\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b7fee661-4a32-425e-b42c-6f4efcb2eec8",
+   "metadata": {},
+   "source": [
+    "## Configuration and Client setup\n",
+    "\n",
+    "In order to connect to your Unity Catalog server, you'll need an instance of the `ApiClient` from the `unitycatalog-client` package. \n",
+    "\n",
+    "> Note: If you don't already have a Catalog and a Schema created, be sure to create them before running this notebook and adjust the `CATALOG` and `SCHEMA` variables below to suit."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "eba0f1bf-ef95-420e-91cd-392e0a4a509f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from unitycatalog.ai.anthropic.toolkit import UCFunctionToolkit\n",
+    "from unitycatalog.ai.anthropic.utils import generate_tool_call_messages\n",
+    "from unitycatalog.ai.core.oss import UnitycatalogFunctionClient\n",
+    "from unitycatalog.client import ApiClient, Configuration"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "9a93a22f-4d55-428e-8576-ee11be1c009d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "config = Configuration()\n",
+    "config.host = \"http://localhost:8080/api/2.1/unity-catalog\"\n",
+    "\n",
+    "# The base ApiClient is async\n",
+    "api_client = ApiClient(configuration=config)\n",
```

Comment:
As a full executable notebook, I think we should include catalog/schema creation step

Reply:
Agreed. I think I'm going to create those utility methods for create-only if not exists for catalogs and schemas within our client API, then we can (in that PR) update the notebooks and examples to use those APIs :)